docker/package/systemd/tezos-baker.service

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+[Unit]
+After=network.target
+After=tezos-node.service
+Requires=tezos-node.service
```

Comment:
This kind of prevents running it against a remote node without running a local one, is this desired?

Reply:
AFAIU, Baker is supposed to be run only with local node